<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveQML</class>
 <widget class="QDialog" name="SaveQML">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>292</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Save styles to qml files</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Layers:</string>
     </property>
     <property name="buddy">
      <cstring>lvMapLayers</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListView" name="lvMapLayers">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pbnApplyStyle">
       <property name="text">
        <string>Save style to QML</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbnSelectAllLayers">
       <property name="text">
        <string>Select all layers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbnAbout">
       <property name="text">
        <string>About</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pbnSelectAllLayers</tabstop>
  <tabstop>lvMapLayers</tabstop>
  <tabstop>pbnApplyStyle</tabstop>
  <tabstop>pbnClose</tabstop>
  <tabstop>pbnAbout</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
